csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -465,11 +465,11 @@ Acknowledgement (`acknowledgement_t`), Branch Branches (`branch_branches_t`), Fu
     },
 
 ### 3.1.1 Acknowledgment Type
```

Comment:
Should we then use plural here since we are talking about the "List of Acknowledgments" which contains the 'Acknowledgment' element or is this nesting confusing?